harmoni_detectors/harmoni_stt/test/deepspeech.test

```diff
@@ -0,0 +1,7 @@
+<launch>
+    <rosparam file="$(find harmoni_stt)/config/deepspeech_configuration.yaml" subst_value="True"/>
+	<param name="instance_id" value="default"/>
+	<node pkg="harmoni_stt" type="deepspeech_service.py" name="harmoni_stt_deepspeech_default" output="screen"/>
+	<param name="test_deepspeech_input" value="$(find harmoni_stt)/test_data/hello_16k.wav"/>
+    <test test-name="test_deepspeech" pkg="harmoni_stt" type="test_deepspeech.py"/>
```

Comment:
It prints that the test succeeded , however the transcription of the chunk audio is empty.

Reply:
The actual transcription functionality is tested in the Python unit test (`test_deepspeech_client.py`), though I can add it to the rostest as well.